src/components/Alert/Alert.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react";
+// import { useEffect } fr/om "react/cjs/react.production.min";
+import { useAlert } from "../../context";
+
+const Alert = ({ alert }) => {
+  const { alertList, setAlertList } = useAlert();
+
+  useEffect(() => {
+    setTimeout(() => {
+      setAlertList((prev) => {
+        return prev.filter((item) => item.id !== alert.id);
+      });
+    }, 2000);
+  }, [alertList]);
+
+  return (
+    <div class={`alert alert-${alert.type}`}>
+      <i
+        class={`fas alert-icon ${
+          alert.type == "success" ? "fa-check-circle" : ""
+        } ${alert.type == "error" ? "fa-exclamation-triangle" : ""} `}
```

Comment:
Suggestion you should make a function and use the switch in it for this, as it will expand with more types of alerts
like this:
```
const getIcon = (type) => {
    switch (type) {
      case "success":
        return "fa-circle-check";
      case "error":
        return "fa-exclamation-triangle"
    }
  };
```